distribution/lib/Standard/Table/0.0.0-dev/src/Data/Type/Value_Type.enso

```diff
@@ -173,6 +173,18 @@ type Value_Type
         Value_Type.Integer _ -> True
         _ -> False
 
+    ## Checks if the `Value_Type` represents a floating-point type.
+    is_floating : Boolean
+    is_floating self = case self of
```

Comment:
This is very easy to confuse with `is_floating_point`. From the method name it is impossible to tell what is the difference between the two.

Maybe `is_float`?